src/typing/DRMTypes.tsx

```diff
@@ -0,0 +1,19 @@
+export enum DRMType {
+    PLAYREADY="com.microsoft.playready",
+    WIDEVINE="com.widevine.alpha",
+    FAIRPLAY="com.apple.fps.1_0",
+    UNKNOWN="unknown",
+}
+
+export function drmTypeFrom(drmType: string): DRMType {
+    switch(drmType.toUpperCase()) {
+        case "PLAYREADY":
+            return DRMType.PLAYREADY;
+        case "WIDEVINE":
+            return DRMType.WIDEVINE;
+        case "FAIRPLAY":
+            return DRMType.FAIRPLAY;
+        default:
+            return DRMType.UNKNOWN
+    }
+}
```

Comment:
Maybe convert this to something like
`
export const drmTypeFrom = (drmType: string): DRMType => {
    const retType = DRMType[drmType];
    return retType;
};
`
Maybe also need to convert the enum to an array. 
But this is just nitpicking